StortSpelProjekt/Game/src/Gamefiles/Components/UpgradeComponents/UpgradeComponent.cpp

```diff
@@ -0,0 +1,86 @@
+#include "UpgradeComponent.h"
+#include "stdafx.h"
+
+component::UpgradeComponent::UpgradeComponent(Entity* parent)
```

Comment:
This class should not inherit from component. If it does, it means that it can be added by itself in main, which I dont think is the purpose of this component?